EndlessClient/Dialogs/GuildDialog.cs

```diff
@@ -29,6 +28,11 @@ private enum GuildDialogState
         private readonly ILocalizedStringFinder _localizedStringFinder;
         private readonly ICharacterProvider _characterProvider;
         private readonly IEOMessageBoxFactory _messageBoxFactory;
+        private readonly IGuildSessionProvider _guildSessionProvider;
+        private readonly IGuildActions _guildActions;
+        private readonly ITextInputDialogFactory _textInputDialogFactory;
+
+        private string _guildDescription { get; set; }
```

Comment:
This should be a private field, not a property

Reply:
Field removed